### PR TITLE
Fix sdk commands to work with the latest SDK version

### PR DESCRIPTION
**Pull request by orgaizer**

### CODE REVIEW

File: webauthn-for-logged-in-users/pages/login.html [hunk 1]
```diff
@@ -13,7 +13,7 @@
     <script type="module" src="./init.js"></script>
     <script
       type="text/javascript"
-      src="https://cdn.identity.security/authentication-sdk-v2/2.10.1-851b0b3.0/TsAuthSdk.min.js"
+      src="https://platform-websdk.idsec-dev.com/platform-websdk/1.4.1-alpha-5521204345.0/ts-platform-websdk.js"
```
Comment (ts-ron) [inline webauthn-for-logged-in-users/pages/login.html:16]: intentionally using a staging SDK? Not sure this is available to external users

Comment (ts-ron) [reply webauthn-for-logged-in-users/pages/login.html:16]: Also don't you prefer to provide the sample with the `/latest/` CDN link?

File: webauthn-for-logged-out-users/pages/home.html [hunk 2]
```diff
@@ -11,7 +11,7 @@
     <title>ACME</title>
     <script
       type="text/javascript"
-      src="https://cdn.identity.security/authentication-sdk-v2/2.10.1-851b0b3.0/TsAuthSdk.min.js"
+      src="https://platform-websdk.idsec-dev.com/platform-websdk/1.4.1-alpha-5521204345.0/ts-platform-websdk.js"
```
Comment (ts-ron) [inline webauthn-for-logged-out-users/pages/home.html:14]: See comment above re staging SDK link

File: webauthn-for-logged-out-users/pages/login.html [hunk 3]
```diff
@@ -12,14 +12,17 @@
     <title>ACME</title>
     <script
       type="text/javascript"
-      src="https://cdn.identity.security/authentication-sdk-v2/2.10.1-851b0b3.0/TsAuthSdk.min.js"
+      src="https://platform-websdk.idsec-dev.com/platform-websdk/1.4.1-alpha-5521204345.0/ts-platform-websdk.js"
```
Comment (ts-ron) [inline webauthn-for-logged-out-users/pages/login.html:15]: Here as well

File: webauthn-for-logged-out-users/pages/register.html [hunk 4]
```diff
@@ -12,14 +12,17 @@
     <title>ACME</title>
     <script
       type="text/javascript"
-      src="https://cdn.identity.security/authentication-sdk-v2/2.10.1-851b0b3.0/TsAuthSdk.min.js"
+      src="https://platform-websdk.idsec-dev.com/platform-websdk/1.4.1-alpha-5521204345.0/ts-platform-websdk.js"
```
Comment (ts-ron) [inline webauthn-for-logged-out-users/pages/register.html:15]: and here

File: common.sh [hunk 5]
```diff
@@ -81,8 +81,8 @@ options=(
     "SAML IDP"
     "WebAuthn Passkey Authentication (autocomplete)"
     "WebAuthn Cross Device"
-    "WebAuthn Internal IDP Users"
-    "WebAuthn External IDP Users"
+    "WebAuthn Logged In Users"
```
Comment (ts-ron) [inline common.sh:84]: Is this the official name we are giving for this feature? `Logged in ` vs `Logged out`?

File: webauthn-for-logged-in-users/pages/login.html (hunk 1 above)
Comment (orgaizer) [reply webauthn-for-logged-in-users/pages/login.html:16]: 1. I will change it to prod before merging, waiting for the SDK PR to be merged
2. Nope, our customers won't use `latest` and we don't want to imply anywhere it's suggested

File: common.sh (hunk 5 above)
Comment (orgaizer) [reply common.sh:84]: yea

File: webauthn-for-logged-in-users/pages/login.html (hunk 1 above)
Comment (ts-ron) [reply webauthn-for-logged-in-users/pages/login.html:16]: Thanks

**Review by ts-ron — APPROVED** [review]

**Pull request merged**